WcaOnRails/app/models/competition.rb

```diff
@@ -48,6 +51,7 @@ class Competition < ActiveRecord::Base
     cellName
     showAtAll
     isConfirmed
+    eventSpecs
```

Comment:
Oh, smart trick! Can you create an issue for that, and add a comment here linking to that issue?